### PR TITLE
Mount pvc

### PR DESCRIPTION
**Pull request by kkasravi**
@inc0 I took your PR and made a few modifications so it would pass the presubmit tests

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/503)

### CODE REVIEW

**Comment by kkasravi** [comment]
@inc0 this PR will require your approval as well per the note about multiple contributors in a PR.

**Comment by inc0** [comment]
thanks @kkasravi I appreciate it ! Although it's not typically good to create new PR based on unmerged PR like mine. Better approach would be to point my mistakes in review and allow me to fix it :) Do you mind if I take your change and include into my PR and include your commit there?

If you rather have it merged here then no problem with me!

/lgtm

**Comment by jlewi** [comment]
@kkasravi @inc0 are we submitting this PR or #469

**Comment by kkasravi** [comment]
@inc0 - Jeremy made some additional changes that help in isolating changes. Next time I'll push a PR to your branch which would allow you to merge fixes into your PR. My apologies.

@jlewi I rebased from master which included your changes and @inc0's original PR.

File: kubeflow/core/tests/jupyterhub_test.jsonnet [hunk 1]
```diff
@@ -130,6 +131,12 @@ std.assertEqual(jupyterhub.parts(params.namespace).jupyterHub(params.jupyterHubI
                      "-f",
                      "/etc/config/jupyterhub_config.py"
                   ],
+                  "env": [
+                     {
+                        "name": "NOTEBOOK_PVC_MOUNT",
+                        "value": "/home/jovyan/work"
```
Comment (jlewi) [inline kubeflow/core/tests/jupyterhub_test.jsonnet]: shouldn't the value be params.jupyterNotebookPVCMount?

**Comment by kkasravi** [comment]
@jlewi updated to use params.jupyterNotebookPVCMount

File: kubeflow/core/tests/jupyterhub_test.jsonnet (hunk 1 above)
Comment (inc0) [reply kubeflow/core/tests/jupyterhub_test.jsonnet]: It's test validation and this path is default param.

**Comment by inc0** [comment]
let's work on this one because it's further in dev.

**Comment by kkasravi** [comment]
@inc0 please add or change as you see appropriate for the mount pvc tests. For example the params could be changed.

**Comment by inc0** [comment]
@kkasravi I think it's good as it is. Unless we get other feedback, it should be good to merge as soon as we'll get your CLA issue fixed:)

**Comment by kkasravi** [comment]
thanks @inc0 - i think it wants another approver and is suggesting @jlewi although I already assigned him. I'll repeat the assignment

**Comment by jlewi** [comment]
Going to manually merge because of the CLA check.

**Pull request merged**